src/main/java/eu/xenit/alfred/content/gateway/GatewayApplication.java

```diff
@@ -179,6 +186,8 @@ public SecurityWebFilterChain springWebFilterChain(
             http.formLogin();
         }
 
+        http.cors(cors -> cors.configurationSource(exchange -> new CorsConfiguration().applyPermitDefaultValues()));
```

Comment:
This was for testing, should be removed.
(Integration tests DO succeed without CORS config)

Reply:
Removed